R/rmdindex.R

```diff
@@ -31,8 +31,8 @@ writeRmdIndex <- function(appName, appDir) {
 }
 
 redirectWebPage <- function(appFile) {
-  meta <- paste('<meta http-equiv="refresh" content="0;',
-                htmlEscape(appFile), '">', sep = "")
+  meta <- paste("<meta http-equiv='refresh' content='0;",
```

Comment:
This file was the only place where some of the quoted strings needed adjustment, as they contained embedded quotes. In all cases, the embedded double-quotes became embedded single-quotes.